ext.php

```diff
@@ -28,7 +28,31 @@ class ext extends \phpbb\extension\base
 	 */
 	public function is_enableable()
 	{
-		return phpbb_version_compare(PHPBB_VERSION, '3.1.4', '>=') && class_exists('ZipArchive');
+		return class_exists('ZipArchive') && ($this->phpbb_31_compatible() || $this->phpbb_32_compatible());
+	}
+
+	/**
+	 * Check phpBB 3.1 compatibility
+	 *
+	 * Requires phpBB 3.1.4 or greater
+	 *
+	 * @return bool
+	 */
+	protected function phpbb_31_compatible()
+	{
+		return phpbb_version_compare(PHPBB_VERSION, '3.1.4', '>=') && phpbb_version_compare(PHPBB_VERSION, '3.2.0-dev', '<');
+	}
+
+	/**
+	 * Check phpBB 3.2 compatibility
+	 *
+	 * Requires phpBB 3.2.0-b3 or greater
+	 *
+	 * @return bool
+	 */
+	protected function phpbb_32_compatible()
```

Comment:
`phpbb_current_compatibility` or later etc.

Even if you cannot change the method name, the first line of the docblock does nothing to prevent confusion